Doc/library/runpy.rst

```diff
@@ -106,15 +106,16 @@ The :mod:`runpy` module provides two functions:
    Execute the code at the named filesystem location and return the resulting
    module globals dictionary. As with a script name supplied to the CPython
    command line, the supplied path may refer to a Python source file, a
-   compiled bytecode file or a valid sys.path entry containing a ``__main__``
-   module (e.g. a zipfile containing a top-level ``__main__.py`` file).
+   compiled bytecode file or a valid :data:`sys.path` entry containing a
+   ``__main__`` module
+   (e.g. a zipfile containing a top-level ``__main__.py`` file).
 
    For a simple script, the specified code is simply executed in a fresh
-   module namespace. For a valid sys.path entry (typically a zipfile or
-   directory), the entry is first added to the beginning of ``sys.path``. The
-   function then looks for and executes a :mod:`__main__` module using the
+   module namespace. For a valid :data:`sys.path` entry (typically a zipfile or
+   directory), the entry is first added to the beginning of :data:`sys.path`.
+   The function then looks for and executes a :mod:`__main__` module using the
```

Comment:
We normally like to keep a maximum of one link to the same thing per paragraph, as too many links can just be distracting for the reader. You already link to `sys.path` on the first mention of it in this paragraph, so I think the changes you've made here to the second and third mention can be reverted:

```suggestion
   directory), the entry is first added to the beginning of ``sys.path``. The
   function then looks for and executes a :mod:`__main__` module using the
```